modules/serverless-rn-1-33-0.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-33-0_{context}"]
+= Red Hat {ServerlessProductName} 1.33
+
+{ServerlessProductName} 1.33 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-33-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.12.
+* {ServerlessProductName} now uses Knative Eventing 1.12.
+* {ServerlessProductName} now uses Kourier 1.12.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.12.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
+* The `kn func` CLI plugin now uses `func` 1.13.
+
+* When specifying Kourier for Ingress and using `DomainMapping`, the TLS for OpenShift Route is set to passthrough, and TLS is handled by the Kourier Gateway. Beginning with Serverless 1.31, you can specify the enabled cipher suites on the side of the Kourier Gateway.
+
+* The `NamespacedKafka` annotation is now deprecated. Use the standard Kafka broker without data plane isolation instead.
+
+* When enabling the automatic `EventType` auto-creation, you can now easily discover events available within the cluster. This functionality is available as a developer preview 
+
+* The `kn` event plugin for the Knative CLI (kn) is now generally available. The `kn` event plugin lets you send events directly from your command line to various destinations, streamlining your development workflow.
```

Comment:
The GA state isn't yet reached. Here's the milestone that tracks it: https://github.com/knative-extensions/kn-plugin-event/milestone/2 and a Jira epic: https://issues.redhat.com/browse/SRVCLI-391